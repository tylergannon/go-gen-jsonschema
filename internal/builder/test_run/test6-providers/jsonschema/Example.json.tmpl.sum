1717758a227c4a91