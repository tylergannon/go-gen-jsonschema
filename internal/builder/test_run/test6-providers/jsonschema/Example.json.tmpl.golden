{"type":"object","properties":{"a":{{.a}},"b":{{.b}},"c":{{.c}}},"required":["a","b","c"],"additionalProperties":false}
